content/providers/01-ai-sdk-providers/08-amazon-bedrock.mdx

```diff
@@ -434,6 +436,21 @@ const { text, reasoning, reasoningDetails } = await generateText({
   },
 });
 
+console.log(reasoning); // reasoning text
+console.log(reasoningDetails); // reasoning details including redacted reasoning
+console.log(text); // text response
+
+// Nova 2 example
+const { text, reasoning, reasoningDetails } = await generateText({
```

Comment:
can you use different variables, since the same const cannot be redefined? Maybe use `result` and `result2` and then use `result.text` etc.